cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/Scheduler.java

```diff
@@ -45,6 +46,18 @@ public void schedule(Id.Program program, SchedulableProgramType programType, Sch
    *
    * @param program Program that needs to be run.
    * @param programType type of program.
+   * @param schedule Schedule with which the program runs.
+   * @param properties properties to be passed to the schedule
```

Comment:
These are only system properties for the Schedule, which are different from the properties that are associated with the Schedule when its created ,right?